EIPS/EIP-720

```diff
@@ -0,0 +1,59 @@
+---
+eip: 720
+title: A Simple Non-Fungible Token Standard
+description: <Description is one full (short) sentence>
+author: Simon Tian <simon@dtopia.me>
+discussions-to: https://github.com/ethereum/EIPs/issues/3753
+status: Draft
+type: Standards Track,
+category (*only required for Standards Track): ERC
+created: 2021-08-21
+requires (*optional): <EIP number(s)>
+replaces (*optional): <EIP number(s)>
+---
+
+## Abstract
+The current ERC721 NFT standard has gained a lot of attention due to its ability to be associated with external contents via a data field `URI`. However, this is also handicaping people from thinking of NFTs not having any external associations. This 
+stardard is going to reduce the ERC721 standard by removing the `URI` data field and introduce a naked NFT that is ideal for representing abstract ownership such as rights. This is particularly useful when a layer of abstraction can be built on top of it.
+
+## Motivation
+In developing dtopia.me, NFTs are used as symbols of ownership of certain utilities that are built upon ERC721 NFTs. This introduces difficulty in explaining what a NFT without `URI` can do to people who are deeply rooted in the current understanding of NFTs.
+A new standard for this type of NFTs must be developed to separate itself from the NFTs that are associated with external contents.
+
+## Specification
+This standard is to remove all data fields as well as functions related to `URI` from ERC721 implementation.
+
+```
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+interface ERC720 {
+    function balanceOf(address owner) external view returns (uint256);
+    function ownerOf(uint256 tokenId) external view returns (address);
+    function name() external view returns (string memory);
+    function symbol() external view returns (string memory);
+    function totalSupply() external view returns (uint256);
+    function approve(address to, uint256 tokenId) external;
+    function getApproved(uint256 tokenId) external view returns (address);
+    function setApprovalForAll(address operator, bool approved) external;
+    function isApprovedForAll(address owner, address operator) external view returns (bool);
+    function transferFrom(address from, address to, uint256 tokenId) external;
+    function safeTransferFrom(address from, address to, uint256 tokenId) external;
+    function safeTransferFrom(address from, address to, uint256 tokenId, bytes memory _data) external;
+}
+```
+## Rationale
+NFTs do not have to be associated with any external metadata or links to media contents. They can exist as standalone abstract atomic tokens. Such tokens allow layers of utilities built on top of them to open up new possibilities.
+Such implementation can be seen in projects like dtopia.me.
```

Comment:
```suggestion
```
EIPs are technical specifications.  We should not be referencing external projects from them.